src/gsy_e/models/myco_matcher/myco_external_matcher.py

```diff
@@ -84,19 +84,25 @@ def publish_orders(self, message):
             for market in area_data["markets"]:
                 self.area_markets_mapping.update(
                     {f"{area_uuid}-{market.time_slot_str}": market})
-                market_offers_bids_list_mapping[area_uuid] = self._get_orders(market, filters)
+                if area_uuid not in market_orders_list_mapping:
+                    market_orders_list_mapping[area_uuid] = {}
+                market_orders_list_mapping[area_uuid].update(
+                    self._get_orders(market, filters))
```

Comment:
The problem was that we were re-assigning the value of this key in each iteration instead of updating the content of the dictionary to add new time slot data
This was the direct cause of the failing ci tests in the myco sdk